crates/uv-static/src/env_vars.rs

```diff
@@ -614,4 +614,15 @@ impl EnvVars {
 
     /// Enables fetching files stored in Git LFS when installing a package from a Git repository.
     pub const UV_GIT_LFS: &'static str = "UV_GIT_LFS";
+
+    /// Number of times that uv run has recursively invoked itself. Used to
```

Comment:
This is implemented mostly be squinting at what already exists and trying to follow the existing pattern. Happy to adjust as needed.